src/ContentBmwEncoding.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023-2024 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { html } from "htm/preact";
+import { BlissSymbol } from "./BlissSymbol";
+import { usePaletteState } from "./GlobalData";
+import { ContentBmwEncodingType } from "./index.d";
+import "./ContentBmwEncoding.scss";
+
+type ContentBmwEncodingProps = {
+  id: string,
+  options: ContentBmwEncodingType
+}
+
+export function ContentBmwEncoding (props: ContentBmwEncodingProps) {
+  const paletteState = usePaletteState();
+  const fullEncoding = paletteState?.fullEncoding ? paletteState?.fullEncoding : [];
+
+  const { id, options } = props;
+  const { columnStart, columnSpan, rowStart, rowSpan } = options;
+
+  // TODO: use a common utility function to calculate the grid position
```

Comment:
Use `getGridStyle()` here since it's available, but see my comment elsewhere about the name of `getGridStyle()`.